src/transport/PairingSession.h

```diff
@@ -77,6 +83,14 @@ class DLL_EXPORT PairingSession
     virtual const char * GetI2RSessionInfo() const = 0;
 
     virtual const char * GetR2ISessionInfo() const = 0;
+
+private:
+    NodeId mPeerNodeId = kUndefinedNodeId;
+    // TODO(#8206): Remove address and use peer cache instead.
+    Transport::PeerAddress mPeerAddress     = Transport::PeerAddress::Uninitialized();
+    static constexpr uint16_t kInvalidKeyId = UINT16_MAX;
```

Comment:
Also, that's `kAnyKeyId` already.  It's pretty confusing to have different parts of the code attach different meaning to the same value....  Probably better to either use `kAnyKeyId` explicitly as an "invalid" marker or to use a different value and ensure we know what our reserved key id values are...

Reply:
Added a TODO. LocalKeyId must be initialized on construction, PeerKeyId is replaced with an optional.